warrior/Actions/CommonActions/common_actions.py

```diff
@@ -198,6 +200,16 @@ def verify_key_already_exists_and_update(orig_dict, new_dict):
                 filepath = getAbsPath(filepath, os.path.dirname(testcasefile_path))
                 with open(filepath, "r") as json_handle:
                     json_doc = json.load(json_handle)
+                    #if slurp is set to True, all content of given json file will be
+                    #store in data repository
+                    if slurp:
+                        print_info("slurp mode enabled, all content of given"+ \
+                            " json file will be stored in data repository")
+                        update_datarepository(json_doc)
+                        print_info("{0} dictionary stored in Warrior data_repository".\
+                            format(json_doc))
+                        status = True
+                        return status 
```

Comment:
Please avoid multiple return statements.

Reply:
used just one return, I have to write a new keyword otherwise, instead of making changes in existing  keyword

Reply:
You can change the ``` if ``` statement o line 215 to an ``` elif ``` and remove the ``` return ``` on line 214

Reply:
done